specification/security/resource-manager/Microsoft.Security/preview/2024-05-01-preview/alerts.json

```diff
@@ -0,0 +1,1246 @@
+        },
+        "extendedLinks": {
+          "readOnly": true,
+          "type": "array",
+          "description": "Links related to the alert",
+          "items": {
+            "$ref": "#/definitions/AlertExtendedLinks"
+          },
+          "x-ms-identifiers": []
+        },
+        "alertUri": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A direct link to the alert page in Azure Portal."
+        },
+        "timeGeneratedUtc": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The UTC time the alert was generated in ISO8601 format."
+        },
+        "productName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the product which published this alert (Microsoft Sentinel, Microsoft Defender for Identity, Microsoft Defender for Endpoint, Microsoft Defender for Office, Microsoft Defender for Cloud Apps, and so on)."
+        },
+        "processingEndTimeUtc": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The UTC processing end time of the alert in ISO8601 format."
+        },
+        "entities": {
+          "readOnly": true,
+          "type": "array",
+          "description": "A list of entities related to the alert.",
+          "items": {
+            "$ref": "#/definitions/AlertEntity"
+          },
+          "x-ms-identifiers": []
+        },
+        "isIncident": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "This field determines whether the alert is an incident (a compound grouping of several alerts) or a single alert."
+        },
+        "correlationKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Key for corelating related alerts. Alerts with the same correlation key considered to be related."
+        },
+        "extendedProperties": {
+          "$ref": "#/definitions/AlertExtendedProperties",
+          "description": "Custom properties for the alert."
+        },
+        "compromisedEntity": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the resource most related to this alert."
+        },
+        "techniques": {
+          "readOnly": true,
+          "type": "array",
+          "description": "kill chain related techniques behind the alert.",
+          "items": {
+            "readOnly": true,
+            "type": "string"
+          }
+        },
+        "subTechniques": {
+          "readOnly": true,
+          "type": "array",
+          "description": "Kill chain related sub-techniques behind the alert.",
+          "items": {
+            "readOnly": true,
+            "type": "string"
+          }
+        },
+        "supportingEvidence": {
+          "type": "object",
+          "additionalProperties": {},
```

Comment:
>additionalProperties": {} [](http://example.com/codeflow?start=11&length=25)

Same here.

Reply:
done

Reply:
needs to be free-form object - otherwise it fails the model validation (and rightfully so).
You can recreate it by running
`oav validate-example .\specification\security\resource-manager\Microsoft.Security\preview\2024-05-01-preview\alerts.json`